docs/vanilla.md

```diff
@@ -0,0 +1,308 @@
+# Velero Plugin for vSphere in Vanilla Kubernetes Cluster
+
+## Table of Contents
+1. [Compatibility](#compatibility)
+2. [Prerequisites](#prerequisites)
+3. [Install](#install)
+4. [Uninstall](#uninstall)
+5. [Backup](#backup)
+6. [Restore](#restore)
+
+## Compatibility
+| vSphere Version         | vSphere CSI Version                                       | Kubernetes Version | Velero Version    | Velero Plugin for vSphere Version |
+|-------------------------|-----------------------------------------------------------|--------------------|-------------------|-----------------------------------|
+| vSphere 6.7U3           | v1.0.2 (update to 1.0.3 when available)                   | v1.14 and higher   | v1.5.1 and higher | v1.1.0 and higher                 |
+| vSphere 6.7U3           | v2.0.1 and higher                                         | v1.17 and higher   | v1.5.1 and higher | v1.1.0 and higher                 |
+| vSphere 7.0 and higher  | CSI 1.0.2 driver features continue to work on vSphere 7.0 | v1.14 and higher   | v1.5.1 and higher | v1.1.0 and higher                 |
```

Comment:
CSI 1.0.2 driver features continue to work on vSphere 7.0 -> CSI driver bundled with vSphere

Reply:
CSI driver will not be bundled with vSphere in vanilla cluster, right?

Reply:
Sorry, I got myself confused:)